src/Cassandra/Data/Linq/CqlCommand.cs

```diff
@@ -155,11 +155,7 @@ public async Task<RowSet> ExecuteAsync()
         {
             object[] values;
             var cqlQuery = GetCql(out values);
-            var session = GetTable().GetSession();
-            var stmt = await _statementFactory.GetStatementAsync(session, Cql.New(cqlQuery, values))
-                                              .ConfigureAwait(false);
-            this.CopyQueryPropertiesTo(stmt);
-            var rs = await session.ExecuteAsync(stmt).ConfigureAwait(false);
+            var rs = await this.SendQuery(cqlQuery, values).ConfigureAwait(false);
```

Comment:
I must use this because `SendQuery` is an extension method for `IInternalSession` interface. Unfortunately, there is no common ancestor for classes which uses this extension so it's not easy to move inside some class.
Of course, we can create a wrapper for this operation (like `CqlCommanSender`) but it sounds a little bit awkward for me.